src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsalertsapi/entity/event/AlertEvent.kt

```diff
@@ -177,6 +180,8 @@ data class AlertsMergedEvent(
       ),
       description = ALERTS_MERGED.description,
       occurredAt = occurredAt.toZoneDateTime(),
+      detailUrl = "$baseUrl/prisoners/$prisonNumberMergeTo/alerts?size=2147483647",
```

Comment:
The alerts merged event will get deleted but I'm guessing these are there to be compatible with the base